main.gms

```diff
@@ -1547,8 +1547,8 @@ $setglobal cm_eni  off  !! def = off
 $setglobal cm_enb  off  !! def = off
 ***  cm_incolearn "change floor investment cost value"
 ***   Example on how to use:
-***     cm_incolearn  "wind=1600,spv=5160,csp=9500"
-***       floor investment costs from learning set to 17000 for EVs; and 1600, 5160 and 9500 for wind, solar pv and solar csp respectively.
+***     cm_incolearn  "windon=1600,spv=5160,csp=9500"
+***       floor investment costs from learning set to 17000 for EVs; and 1600, 5160 and 9500 for windon, solar pv and solar csp respectively.
```

Comment:
It would be very nice if you could briefly remove "17000 for EVs; and" - it is not in the line above anymore (the person removing EVs apparently forgot to update the comment)